workspaces/lightspeed/plugins/lightspeed/.prettierrc.js

```diff
@@ -0,0 +1,20 @@
+// @ts-check
+
+/** @type {import("@ianvs/prettier-plugin-sort-imports").PrettierConfig} */
+module.exports = {
+  ...require('@spotify/prettier-config'),
+  plugins: ['@ianvs/prettier-plugin-sort-imports'],
+  importOrder: [
+    '^react(.*)$',
+    '',
+    '^@backstage/(.*)$',
+    '',
+    '<THIRD_PARTY_MODULES>',
+    '',
+    '^@janus-idp/(.*)$',
```

Comment:
`@backstage-community/`?

Reply:
both frontend and backend updated to`red-hat-developer-hub`